cog.c

```diff
@@ -253,20 +253,18 @@ on_handle_local_options (GApplication *application,
             return EXIT_FAILURE;
         }
 
-        g_object_set (shell, "config-file", g_key_file_ref (key_file), NULL);
```

Comment:
Can you split this PR in 2 commits please? One with the style fixes and one with the deprecation warning fixes.

Reply:
The pre-commit hook prevents you from committing anything if your code format isn't what is required. When I made the modifications on cog.c, I couldn't commit unless the code-style format in this file matched what is required so it might be complicated separating the changes into two commits since one change caused the other.

Reply:
The style fixes on cog-handler.h can be put in another commit but the style fixes on cog.c were made because of the change on the file. The pre-commit hook prevents committing of any change not in the right code format. After I made the change on cog.c, I hard to fix the code-style before the commit was accepted.

Reply:
You can skip the pre-commit hooks with `git commit -n ...`